Skip new subapp caching test over 12 procs while under investigation

GiudGiud opened the pull request:
refs #20962

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131291/) on 4cf2d74 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131291/) on 4cf2d74 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131291/) on 4cf2d74 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/1131311/) on 4cf2d74 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/21782/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/1131321/) on 4cf2d74 wanted to post the following:


## Framework coverage

No coverage report(s) were found for the base commit 7bbc67e40c.
[Full coverage report](https://mooseframework.inl.gov/docs/PRs/21782/coverage/framework)


## Modules coverage

No coverage report(s) were found for the base commit 7bbc67e40c.

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/contact)
- [`electromagnetics`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/electromagnetics)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/richards)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/tensor_mechanics)
- [`thermal_hydraulics`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/thermal_hydraulics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/21782/coverage/xfem)

</details>



This comment will be updated on new commits.

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131426/) on 808053c wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131426/) on 808053c wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131426/) on 808053c wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

GiudGiud commented:
refs #21783

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131712/) on 658ecd7 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131712/) on 658ecd7 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1131712/) on 658ecd7 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

loganharbour reviewed (dismissed):
Add issue detail, make PR description relevant. good otherwise 👍🏼

GiudGiud commented:
The fix in the PR, and the test we added, are valuable in themselves (and do not introduce a new bug, only fix one) so we're reluctant to revert. They will help @muelcm and possibly @aprilnovak in their modeling. The new test is unfortunately uncovering a new bug, so that case with higher MPI count than subapps will be studied next week.

loganharbour committed:
Skip new subapp caching test over 12 procs

refs #21783

This test was added in #21748. The fix in said PR is valuable (fixes a current bug,
introduces no new known bugs), so we're reluctant to revert. This new test uncovers
a new bug and will be addressed separately.

g

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1132126/) on 7bd6232 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1132126/) on 7bd6232 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1132126/) on 7bd6232 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1132451/) on 24924e9 wanted to post the following:

The following file(s) are changed:

  conda/mpich/conda_build_config.yaml

The conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1132451/) on 24924e9 wanted to post the following:

The following file(s) are changed:

  conda/petsc/meta.yaml

The PETSc conda configuration has changed; ping @idaholab/moose-dev-ops

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/1132451/) on 24924e9 wanted to post the following:

The following file(s) are changed:

  conda/libmesh/meta.yaml

The libmesh conda configuration has changed; ping @idaholab/moose-dev-ops
